features/meson.build

```diff
@@ -83,6 +83,10 @@ if feature_data.get('_lib_lchmod', 0) == 0
     endif
 endif
 
+feature_data.set10('_lib_getrusage',
+    cc.has_function('getrusage',
```

Comment:
On which systems this test will fail ?

Reply:
No idea if there are any systems we care about which don't provide `getrusage()`. This is just a bit of extra caution since the code did not use it prior to this change; and instead used the inferior `times()` API.